schemas/extractor.yml

```diff
@@ -189,6 +200,36 @@ enums:
                     parametrised using arguments, shall be supplied in the `command`
                     slot.
 
+    UsageScope:
+        permissible_values:
+            meta-only:
+                description: >-
+                    This usage scope only returns metadata associated with the file,
+                    according
+                    to the extractor's own interpretation of metadata. This may include,
+                    but is not
+                    limited to, timestamps, column headings, data types and data shapes.
+                    The size
+                    of data returned in this mode should be limited and will typically
+                    be much smaller
+                    than the file size.
```

Comment:
Worth mentioning that this ought to be a `json` file (or something implementing the dict interface, so I guess a `Mapping`, when returned in memory), **guaranteed**.